src/Mod/ReverseEngineering/Gui/Command.cpp

```diff
@@ -421,8 +421,8 @@ CmdSegmentation::CmdSegmentation()
 {
     sAppModule = "Reen";
     sGroup = QT_TR_NOOP("Reverse Engineering");
-    sMenuText = QT_TR_NOOP("Mesh segmentation...");
-    sToolTipText = QT_TR_NOOP("Create mesh segments");
+    sMenuText = QT_TR_NOOP("Mesh Segmentation…");
+    sToolTipText = QT_TR_NOOP("Creates mesh segments");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Creates separate mesh segments based on surface types");
```
Added info from [docs](https://wiki.freecad.org/Mesh_Segmentation/en) description